go/vt/sqlparser/ast.go

```diff
@@ -2077,6 +2078,84 @@ type TrimFuncType int8
 // TrimType is an enum to get types of Trim
 type TrimType int8
 
+// Types for window functions
+type (
+
+	// WindowSpecification represents window_spec
+	// More information available here: https://dev.mysql.com/doc/refman/8.0/en/window-functions-usage.html
+	WindowSpecification struct {
+		Name            ColIdent
+		PartitionClause Exprs
+		OrderClause     OrderBy
+		FrameClause     *FrameClause
+	}
+
+	WindowDefinition struct {
+		Name       ColIdent
+		WindowSpec *WindowSpecification
+	}
+
+	WindowDefinitions []*WindowDefinition
+
+	NamedWindow struct {
+		Windows WindowDefinitions
+	}
+
+	NamedWindows []*NamedWindow
+
+	// FrameClause represents frame_clause
+	// More information available here: https://dev.mysql.com/doc/refman/8.0/en/window-functions-frames.html
+	FrameClause struct {
+		Unit      FrameUnitType
+		IsBetween bool
```

Comment:
This bool is not needed. If `End` is not nil, we are dealing with a Between

Reply:
Remove the field.